lang/en/content.php

```diff
@@ -57,4 +57,14 @@
     "about.section_4.subtitle_1" => "Our company promotes ",
     "about.section_4.subtitle_2" => "work-life balance, ",
     "about.section_4.subtitle_3" => "giving employees the opportunity to develop outside of the professional sphere as well. After hours, our team builds bonds through shared passions such as board games, mountain trips, basketball tournaments, and regular outdoor meetings.",
+    "case_studies.section_1.title_1" => "Our ",
+    "case_studies.section_1.title_2" => "projects",
+    "case_studies.section_1.subtitle_1" => "See what we worked on for our clients.",
+    "case_studies.section_1.subtitle_2" => "Each case study contains a detailed description of the project implementation.",
```

Comment:
```suggestion
    "case_studies" => [
        "section_1" => [
            "title_2" => "projects",
            "subtitle_1" => "See what we worked on for our clients.",
            "subtitle_2" => "Each case study contains a detailed description of the project implementation."
        ],
    ],
```
maybe that should be more readable?

Reply:
formatted